### PR TITLE
feat: add 0/1 player smash and remove no contest screen

### PR DESCRIPTION
**Pull request by blu-dev**
If you pick time battle, all players set to random to nothing.

This also requires removing the no contest/result screen in time battles to not soft lock the game. I also removed the no contest in stock battle, it just goes back to CSS when you quit now.

### CODE REVIEW

File: plugin/src/lib.rs [hunk 1]
```diff
@@ -207,7 +216,9 @@ unsafe fn push_something(game_state: u64, amt: u32) {
     let mut val2 = *game_state.add(0x108 / 8);
     let mut val3 = if val2 - val1 != 0 {
         (val2 - val1) * 0x20 - 1
-    } else { 0 };
+    } else {
```
Comment (HisashiASM) [inline plugin/src/lib.rs:219]: Ew, don't like this, but it is whatever

File: plugin/src/lib.rs [hunk 2]
```diff
@@ -294,17 +370,42 @@ pub extern "C" fn main() {
...
+            load_ingame_call_sequence_scene,
+            load_melee_scene,
+            game_end,
+            game_exit
+        );
     }
 
     #[cfg(not(feature = "runtime"))]
-    { utils::init(); }
+    {
```
Comment (HisashiASM) [inline plugin/src/lib.rs:389]: Seems to do it on one liner functions, but whatever (Does Rust support no curly brace on one line functions?)
`#[cfg(not(feature = "runtime"))]
          utils::init();`
or something?

**Review by HisashiASM — DISMISSED** [review]
LGTM

File: plugin/src/lib.rs (hunk 2 above)
Comment (blu-dev) [reply plugin/src/lib.rs:389]: don't think so bad

**Pull request merged**